Reactive: Integration tests with validated endpoints fail because of blocking call

<!--
- Please follow the issue template below for bug reports.
- If you have a support request rather than a bug, please use [Stack Overflow](http://stackoverflow.com/questions/tagged/jhipster) with the JHipster tag.
- For bug reports it is mandatory to run the command `jhipster info` in your project's root folder, and paste the result here.
- Tickets opened without any of these pieces of information will be **closed** without any explanation.
-->

##### **Overview of the issue**

Integration tests of Ressource-Endpoints with `@Valid`-annotated method arguments fail, because BlockHound detects a blocking call. The call is made by `SpringValidatorAdapter`.

The error message is:
`reactor.blockhound.BlockingOperationError: Blocking call! java.io.RandomAccessFile#readBytes
	at java.base/java.io.RandomAccessFile.readBytes(RandomAccessFile.java)`

##### **Related issues**
#7608, [#24247](https://github.com/spring-projects/spring-framework/issues/24247)

##### **How to reproduce**
Run `ExceptionTranslatorIT.testMethodArgumentNotValid`


##### **Suggest a Fix**

A workaround is to disable BlockHound in the class `JHipsterBlockHoundIntegration` by adding
`builder.allowBlockingCallsInside("org.springframework.validation.beanvalidation.SpringValidatorAdapter", "validate");`

The linked issues suggest that the problem is already fixed (via --parameters). I am using the spring-boot gradle-plugin which should set this automatically and still get the error.

Is it only me having this?

##### **JHipster Version(s)**
7.0.0-beta.1


- [x] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

I created an app with the current `main` branch and your `.yo-rc.json`. Then, I ran:

```
./gradlew test integrationTest
```

I can confirm this is indeed an issue.

```
> Task :compileJava
Note: Hibernate JPA 2 Static-Metamodel Generator 5.4.29.Final
Note: /Users/mraible/dev/jh-14326/src/main/java/com/mycompany/myapp/security/jwt/JWTFilter.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/mraible/dev/jh-14326/src/main/java/com/mycompany/myapp/security/jwt/TokenProvider.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :integrationTest

ExceptionTranslatorIT > testMethodArgumentNotValid() FAILED
    java.lang.AssertionError at ExceptionTranslatorIT.java:51

9 tests completed, 1 failed

> Task :integrationTest FAILED

FAILURE: Build failed with an exception.
```

I tried adding this:

```
builder.allowBlockingCallsInside("org.springframework.validation.beanvalidation.SpringValidatorAdapter", "validate");
```

And it fixes it! 

```
BUILD SUCCESSFUL in 30s
```

Can you please create a PR with this fix?

--- a/generators/server/templates/src/test/java/package/config/JHipsterBlockHoundIntegration.java.ejs
+++ b/generators/server/templates/src/test/java/package/config/JHipsterBlockHoundIntegration.java.ejs
@@ -27,6 +27,7 @@ public class JHipsterBlockHoundIntegration implements BlockHoundIntegration {
         // Workaround until https://github.com/reactor/reactor-core/issues/2137 is fixed
         builder.allowBlockingCallsInside("reactor.core.scheduler.BoundedElasticScheduler$BoundedState", "dispose");
         builder.allowBlockingCallsInside("reactor.core.scheduler.BoundedElasticScheduler", "schedule");
+        builder.allowBlockingCallsInside("org.springframework.validation.beanvalidation.SpringValidatorAdapter", "validate");
     }
 
 }

